docs/docs/cloud/quick_start.md

```diff
@@ -74,21 +74,21 @@ In order to do this we can first install the LangGraph CLI
 pip install langgraph-cli
 ```
 
-We can then stand up a simple test server. The server this stands up is INCREDIBLY simple - it is just a single endpoint and has no persistence. **This should not be used for hosting your application, only for testing the build and basic functionality.**
+We can then test our API server as long as we have a valid `LANGCHAIN_API_KEY` and have access to LangGraph Cloud closed beta.
```

Comment:
they should have a valid LangChain API key so id make it more like "we can then test our API server locally. this requires access to langGraph closed beta. In order to run the server locally, you will need to add your LANGCHAIN_API_KEY to the .env file so we can validate you have access to LangGraph closed beta"